### ISSUE TITLE
BodyHeatmap SVG muscle regions missing aria-labels on right side (right shoulder, bicep, quad, rear shoulder, tricep, hamstring) while left-side counterparts have them

### ISSUE THREAD

**Issue by aschung212** [aschung212/Lift#131]
No description
